config/sidekiq.yml

```
@@ -1,4 +1,2 @@
 ---
-:queues:
-  - default
-  - ingest
+:queues: <%= ESSI.config.fetch(:sidekiq,{}).fetch(:queue_names,nil) || "\n- default\n- ingest\n" %>
```

An inline array might be clearer.
```suggestion
:queues: <%= ESSI.config.fetch(:sidekiq,{}).fetch(:queue_names,nil) || "['default', 'ingest']" %>
```

---

@dlpierce Requested changes made and pushed.